### PR TITLE
update code to run apf customized workflow

### PR DESCRIPTION
**Pull request by liqcui**
Open a new pr for apf customized workflow since Finn leave our team, I couldn't directly change the code in his PR

### CODE REVIEW

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-loaded-upgrade-from-4.14-perfscale-aws-24nodes-loaded-upgrade-414to415

**Comment by liqcui** [comment]
@paigerube14 @qiliRedHat @svetsa-rh I open a new pr for Finn's task, I couldn't push code to his code repository when I make change for the code. Please review the pr when you are available, thanks!

**Comment by svetsa-rh** [comment]
@liqcui pls remind me as we had this discussion before.. 
instead of running the test as post step for loaded upgrade, can we not run it like a regular workload test step? what issue did we encounter when we tried to do this? I am trying to see how this test can be run as an independent test.

Other than that, code and test results look good to me. Will let @paigerube14 and @qiliRedHat also weigh in.

**Comment by svetsa-rh** [comment]
If no objection from @paigerube14 and @qiliRedHat to tag the APF test after loaded upgrade, let's use this PR to address these as well https://github.com/openshift/release/pull/45646#discussion_r1456514360
If we find better alternative, let's address above comment in a separate PR.

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-conc-builds-3nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-stable-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-stable-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-stable-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-stable-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-stable-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-stable-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-stable-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-stable-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-stable-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-conc-builds-3nodes

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.14-stable-x86-loaded-upgrade-from-4.14-perfscale-rosa-hcp-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade413

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade413

**Comment by liqcui** [comment]
@svetsa-rh @qiliRedHat @paigerube14 I plan to combined apf customized workflow with rosa loaded upgrade code, but rosa loaded upgrade with block issue, so separated the two case into two pr again, please review this pr when you are available, thanks!

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ tests:
     env:
       CHANNEL_GROUP: candidate
       CHURN: "false"
-      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      COMPUTE_MACHINE_TYPE: m5.xlarge
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml:41]: @liqcui If you do this change, I think openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-candidate-x86-loaded-upgrade-from-4.12.yaml should be changed to make them consistent. In rosa-aws-sts workflow the env is set to COMPUTE_MACHINE_TYPE: "m5.2xlarge"

Comment (liqcui) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml:41]: Fixed in new code, thanks!

Comment (qiliRedHat) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml:41]: @liqcui lgtm now. Let's see if @svetsa-rh and @paigerube14 have other comments.

Comment (paigerube14) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml:41]: I think theres' a larger effort to align different tests to have similar compute/master node sizes. I dont think this pr is where we should be updating that. Can we stick to only adding the APF tests and put this into discussion in another pr?

Comment (liqcui) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml:41]: Hi Paige, I can move the change to my another rosa loaded upgrade pr

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-conc-builds-3nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml (hunk 1 above)
Comment (liqcui) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml]: @paigerube14 I have move the change to another rosa loaded upgrade pr, please review again, thanks!

**Pull request merged**